src/kv/rocksdb_cache/BinnedLRUCache.h

```diff
@@ -296,6 +326,20 @@ class alignas(CACHE_LINE_SIZE) BinnedLRUCacheShard : public CacheShard {
   // We don't count mutex_ as the cache's internal state so semantically we
   // don't mind mutex_ invoking the non-const actions.
   mutable std::mutex mutex_;
+
+  // Circular buffer of byte counters for age binning
+  boost::circular_buffer<std::shared_ptr<uint64_t>> age_bins;
```

Comment:
Is this circularity ever used?
We push new element in `rotate_bins` and trim in `import_intervals`.
Maybe it is just a list?